src/libelektra/mount.c

```diff
@@ -444,6 +444,8 @@ Backend* elektraMountGetBackend(KDB *handle, const Key *key)
 {
 	if (!key || !strcmp(keyName(key), "")) return handle->defaultBackend;
 
+	if (!handle) return NULL; // TODO: what is the best way to handle this
```

Comment:
It was happening due to libgetenv for some reason, since libgetenv is disabled for OS X now I guess I can revert that change